README.md

```diff
@@ -1,3 +1,4 @@
+# Colorable
```

Comment:
I think the reason I left this out was because I'm using the README to generate the site content. Would need to strip that out in `docs/data.js` I think...